docs/user-guide/errors.md

```diff
@@ -98,3 +98,42 @@ You should:
 - install the latest version of Stylelint (`npm i --save-dev stylelint`), as we may have recently added the option
 - check that the option exists and is correctly named by reading the rule's README
 - correctly configure the [`rules`](configure.md#rules) property in the configuration object
+
+## Warning with EditInfo
+
+When the `computeEditInfo` option is enabled and `fix` option is disabled, warnings may include a `fix` property that provides information about suggested fixes.
+This is useful for automated fixing tools and IDE integrations.
+
+### EditInfo type
+
+The `EditInfo` value is information to edit text. It is included in warnings' `fix` property. It has following properties:
+
+- `range` (`[number, number]`)
+  The pair of 0-based indices in source code text to remove.
+
+- `text` (`string`)
+  The text to add.
+
+For example, to change `a { color: #FFF; }` to `a { color: #FFFFFF; }`, the EditInfo might look like:
+
+```js
+{
+  // Index for the last "F" in "#FFF"
+  range: [13, 14],
+  // The replacement text
+  text: "FFFF"
+}
+```
```

Comment:
This example happens to include the special casing for pure insertions.
In such cases the range would be `[14, 14]` with text `FFF`, but we extend this so that ranges are never points.

Should we use a different example instead?
I think this could be simpler to get readers familiar with the feature.

Maybe `a { color: #FFF; }` to `a { color: white; }`?

This special case is also something that implementers of tools don't have to handle specifically. It is just an internal implementation detail to have unambiguous ranges. Having unambiguous ranges is important to correctly detect overlap.

Reply:
I see. If the [color-named](https://stylelint.io/user-guide/rules/color-named/) rule supported fixes, that example would be good. 
But since it doesn’t at the moment, I’ll think of another clear example.

Reply:
Fixed it.
[Update example](https://github.com/stylelint/stylelint/pull/8485/commits/11357b2ad9526ae913bb4e78e5cdae550025f174)